lib/view_component/storybook/controls/control_config.rb

```diff
@@ -22,6 +23,13 @@ def name(new_name = nil)
           end
         end
 
+        def description(new_description = nil)
+          return @description if new_description.nil?
+
+          @description = new_description
+          self
+        end
```

Comment:
This looks like what `attr_writer` does, or am I missing something subtle 🤔 

```
class ControlConfig
  attr_writer :description
end

config = ControlConfig.new(description: "Soon to be out of date description")
config.description = "My new description"
config.description #=> "My new description"
```

Reply:
Yeah, the detail is the method returning`self` when you set a description with it so you can chain it when writing stories:

```ruby
class PersonComponentStories < ViewComponent::Storybook::Stories
  story :person do
    constructor(
      first_name: text("Nelson").description("The person's first name").name('Not First Name'),
      last_name:  text("Mandela").description("The person's last name")
    )
  end
end
```